src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/mapper/FlowMapper.java

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.flowmonitoring.mapper;
+
+import org.openkilda.messaging.info.flow.UpdateFlowInfo;
+import org.openkilda.messaging.info.stats.FlowRttStatsData;
+import org.openkilda.messaging.payload.flow.PathNodePayload;
+import org.openkilda.model.Flow;
+import org.openkilda.model.PathSegment;
+import org.openkilda.wfm.topology.flowmonitoring.model.FlowPathLatency;
+import org.openkilda.wfm.topology.flowmonitoring.model.FlowState;
+import org.openkilda.wfm.topology.flowmonitoring.model.Link;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface FlowMapper {
```

Comment:
It seems that the mapper operates with flowpath and segments only. Why not call it FlowPathMapper?